services/galley/test/integration/API/MLS.hs

```diff
@@ -1421,3 +1430,46 @@ testPublicKeys = do
           (unMLSPublicKeys keys)
       )
       @?= [Ed25519]
+
+propUnsupported :: TestM ()
```

Comment:
This is a somewhat unconventional test. Maybe add a couple of lines of comment explaining what it is we're doing here, like the fact that we are manually reading mls-test-cli's store and extracting the private key.